test/fixtures/PeerTestHarness.ts

```diff
@@ -932,6 +933,152 @@ export class PeerTestHarness<T extends AStateMachine> {
         return peer.stateManager.storage.timeout.getTimeout(forkId);
     }
 
+    /**
+     * Creates a real dispute on-chain by having a peer call disputeManager.dispute()
+     * and waits for it to be synced to all peers' local diamonds
+     */
+    async createDispute(peerIndex: number, forkId: ForkId): Promise<void> {
```

Comment:
create dispute based on what?
did any participent author a invalid block?

why is this needed? you can use `submitDoubleSignBlock` or `submitInvalidStateTransitionBlock` which will result in dispute being created